bitnami/kube-state-metrics/values-production.yaml

```diff
@@ -0,0 +1,243 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+##
+global:
+  # imageRegistry: myRegistryName
+  # imagePullSecrets:
+  #   - myRegistryKeySecretName
+  # storageClass: myStorageClass
+
+  labels: {}
+  #   foo: bar
```

Comment:
??

Reply:
added field description: `Additional labels to apply to all resources`